packages/suite/src/components/suite/images/images.ts

```diff
@@ -104,4 +104,5 @@ export const PNG_IMAGES = {
     WALLET: 'Wallet.png',
     WALLET_2x: 'Wallet@2x.png',
     WALLET_3x: 'Wallet@3x.png',
+    TOR_ENABLING: 'Tor_enabeling.png',
```

Comment:
would be nice to rename the file too

Reply:
It should be fix now, thanks for the detail!